secateur/otel.py

```diff
@@ -62,19 +62,22 @@ def preferred_temporality(
         return opentelemetry.sdk._metrics.point.AggregationTemporality.DELTA
 
 
-if os.environ.get("METRICS_EXPORT_ENDPOINT"):
-    metric_exporter = DeltaOTLPMetricExporter(
-        endpoint=os.environ.get("METRICS_EXPORT_ENDPOINT"),
+metric_exporters = []
```

Comment:
through a  _ in front of these variables to make it clear they're "internal" to the module :-)

Reply:
i.e. `_metric_exporters`

Reply:
Sure, done.